### ISSUE TITLE
How do you add metadata on a draft message?

### ISSUE THREAD

**Issue by Eric-Luna** [nylas/nylas-python#205]
**Describe the bug**
I'm trying to add a metadata attr on the draft but it's not appearing on the created webhook

**To Reproduce**
```
draft = self.nylas_client.drafts.create()
...
draft.metadata = {"extra": 1}
draft.send()
```

**SDK Version:**
Version: 5.10.0

**Comment by mrashed-dev** [comment]
Thanks for opening this ticket @Eric-Luna, we are taking a look at this issue!

**Comment by mrashed-dev** [comment]
@Eric-Luna Thanks for opening the issue, the bug is now fixed and is available in the latest [v5.7.0 release](https://github.com/nylas/nylas-python/releases/tag/v5.7.0)!
